packages/connectivity/src/scp-cf/xsuaa-service.ts

```diff
@@ -116,7 +116,7 @@ export async function getClientCredentialsToken(
         null,
         subdomainAndZoneId.zoneId,
         (err: Error, token: string, tokenResponse: ClientCredentialsResponse) =>
-          err ? reject(err) : resolve(tokenResponse)
+          err ? reject(err.message) : resolve(tokenResponse)
```

Comment:
[req] Please add a string in front of the message like `Error in fetching the token for service ${serviceName}: ${error.message}`

Reply:
I chenged the meessage. I added a type predicated and checker that is needed for generating the message as well.